FixedPopupPanels/tests/FixedPopupPanels-specs.js

```diff
@@ -81,4 +111,20 @@ describe('FixedPopupPanels', () => {
 		expect(handleClose).toHaveBeenCalled();
 		expect(actualEvent).toMatchObject(expectedEvent);
 	});
+
+	test('should navigate to the second panel on arrow left key',  function () {
```

Comment:
Let's remove this test case or replace the left key with the right key since this isn't following the UX scenario.

Reply:
Hello, regarding this test case, we've decided to integrate this test case to cover the lines 47-56 from FixedPopupPanels.js. 
![image](https://user-images.githubusercontent.com/111972354/195302931-e3339d9a-95fd-484d-837a-40b888b71284.png)
Removing this test case will decrease the code coverage from 100% to 64.7 as shown in the image below leaving the lines 47-56 uncovered.
![image](https://user-images.githubusercontent.com/111972354/195303490-187c8f8f-9c26-4477-825d-6d5eb82baf96.png)
Also, replacing the left key with the right key will decrease the code coverage from 100% to 70,58% as shown in the image below leaving the lines 50-56 uncovered.
![image](https://user-images.githubusercontent.com/111972354/195303949-19d0861f-75d4-485e-8f64-9a4b6b6bf84c.png)

Reply:
I don't know why the coverage result is different from mine but when I removed the first test you added, the coverage is still 100%.
Would you please check out the result from codecov of my branch here https://app.codecov.io/gh/enactjs/sandstone/tree/feature%2FWRO-12376-mk/FixedPopupPanels?
The difference between yours and mine is https://github.com/enactjs/sandstone/compare/feature/WRO-12376...feature/WRO-12376-mk?expand=1.

The onKeyDown handlers line from 44 to 61 of FixedPopupPanels is exactly for the second test. It should not be affected by the first test.

Would you please take a look again?

Reply:
Hello, the reason why the results were different its because it was my understanding that the replacement of the "left key" should be done for both test cases. Removing/replacing them both with "right key" gave me the results that I presented in my first comment. (my bad)
I removed the first test case and indeed the code coverage is still 100%, everything should be fine now.

Reply:
Oh, okay. Next time, I'll be more specific. Thanks!